tools/build/third_party/breakpad.bzl

```diff
@@ -23,16 +23,16 @@ def _breakpad_impl(repository_ctx):
     )
     repository_ctx.symlink(repository_ctx.attr.build_file, "BUILD")
 
-    if repository_ctx.os.name.startswith("windows"):
-      # Patch up breakpad on windows and add the dump_syms src.
-      repository_ctx.symlink(Label("@gapid//tools/build/third_party/breakpad:windows.patch"), "windows.patch")
-      repository_ctx.symlink(Label("@gapid//tools/build/third_party/breakpad:dump_syms_pe.cc"), "src/tools/windows/dump_syms/dump_syms_pe.cc")
+    # if repository_ctx.os.name.startswith("windows"):
```

Comment:
Do we need a todo or an explanation note for the commented out section here?

Reply:
I will remove this. There used to be a patch but I think the patch is not needed as we updated the version and the issue is solved by the package itself. Thanks for picking!

Reply:
Done!